cfgov/unprocessed/js/organisms/FilterFilterableList.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// Required modules.
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+var Expandable = require( '../molecules/Expandable' );
+
+/**
+ * FilterFilterableList
+ * @class
+ *
+ * @classdesc Initializes a new Filter (Filterable List) organism.
+ *
+ * @param {HTMLNode} element The DOM element within which to search for the organism.
+ */
+function FilterFilterableList( element ) {
+  var BASE_CLASS = 'o-filter-filterable-list';
```

Comment:
Is it worth writing, managing, and loading the extra script? Would it be better to handle that scenario when/if it comes?